data/mapping_Location_v4.json

```diff
@@ -1,409 +1,409 @@
-    { "id": 336,"formId":"0x00017587"},
-    { "id": 337,"formId":"0x00017209"},
-    { "id": 338,"formId":"0x0001735f"},
-    { "id": 339,"formId":"0x0006b588"},
-    { "id": 340,"formId":"0x00017570"},
-    { "id": 341,"formId":"0x00017357"},
-    { "id": 342,"formId":"0x00017584"},
-    { "id": 343,"formId":"0x00017577"},
-    { "id": 344,"formId":"0x000ad3e5"},
-    { "id": 345,"formId":"0x00017564"},
-    { "id": 346,"formId":"0x000a3520"},
-    { "id": 347,"formId":"0x000175a9"},
-    { "id": 348,"formId":"0x000175ad"},
-    { "id": 349,"formId":"0x000175aa"},
-    { "id": 350,"formId":"0x00017597"},
-    { "id": 351,"formId":"0x00017573"},
-    { "id": 352,"formId":"0x00017593"},
-    { "id": 353,"formId":"0x0001735d"},
-    { "id": 354,"formId":"0x00017355"},
-    { "id": 355,"formId":"0x00017560"},
-    { "id": 356,"formId":"0x00017351"},
-    { "id": 357,"formId":"0x0001757c"},
-    { "id": 358,"formId":"0x0001756c"},
-    { "id": 359,"formId":"0x0001756b"},
-    { "id": 360,"formId":"0x0002af0b"},
-    { "id": 361,"formId":"0x0003756b"},
-    { "id": 362,"formId":"0x000177b0"},
-    { "id": 363,"formId":"0x00090c8d"},
-    { "id": 364,"formId":"0x000177ad"},
-    { "id": 365,"formId":"0x000177a5"},
-    { "id": 366,"formId":"0x0003f27f"},
-    { "id": 367,"formId":"0x00028552"},
-    { "id": 368,"formId":"0x0001779d"},
-    { "id": 369,"formId":"0x000ac549"},
-    { "id": 370,"formId":"0x0001779c"},
-    { "id": 371,"formId":"0x000ac544"},
-    { "id": 372,"formId":"0x000177a3"},
-    { "id": 373,"formId":"0x00030027"},
-    { "id": 374,"formId":"0x000177ab"},
-    { "id": 375,"formId":"0x000177ae"},
-    { "id": 376,"formId":"0x00017799"},
-    { "id": 377,"formId":"0x0001779f"},
-    { "id": 378,"formId":"0x000ac548"},
-    { "id": 379,"formId":"0x000177a0"},
-    { "id": 380,"formId":"0x000177a2"},
-    { "id": 381,"formId":"0x000177a1"},
-    { "id": 382,"formId":"0x000908c2"},
-    { "id": 383,"formId":"0x0000ab52"},
-    { "id": 384,"formId":"0x000177a4"},
-    { "id": 385,"formId":"0x000c431b"},
-    { "id": 386,"formId":"0x000cd529"},
-    { "id": 387,"formId":"0x000177a8"},
-    { "id": 388,"formId":"0x000177a6"},
-    { "id": 389,"formId":"0x000177a7"},
-    { "id": 390,"formId":"0x000177b1"},
-    { "id": 391,"formId":"0x0003b369"},
-    { "id": 392,"formId":"0x0003b36b"},
-    { "id": 393,"formId":"0x00097008"},
-    { "id": 394,"formId":"0x0003b366"},
-    { "id": 395,"formId":"0x0003b368"},
-    { "id": 396,"formId":"0x0003b365"},
-    { "id": 397,"formId":"0x0003b367"},
-    { "id": 398,"formId":"0x0003b36c"},
-    { "id": 399,"formId":"0x0003b36a"},
-    { "id": 400,"formId":"0x0003b370"},
-    { "id": 401,"formId":"0x0003b371"},
-    { "id": 402,"formId":"0x0003b36d"},
-    { "id": 403,"formId":"0x0003b363"},
-    { "id": 404,"formId":"0x0003b364"},
-    { "id": 405,"formId":"0x0003b36e"},
-    { "id": 406,"formId":"0x0000cefa"}
-    ]
\ No newline at end of file
+    { "id": 0,"formId":"0x000175fb","tsp":202},
```

Comment:
I used TSP to stand for traveling salesman path.

idk if path communicates it well enough, but we could also switch it out to route/circuit, etc if that communicates the idea better.

Reply:
maybe "tspID" or "tspOrder"?

Reply:
I think tspID works best.